submissions/Eugene-CG/memory-pair-game/index.js

```diff
@@ -0,0 +1,71 @@
+const cardsList = document.querySelector(".cards__list");
+let clickRemember = [];
```

Comment:
Just try to translate this in google translate :) And imagine how it will look like for native speaker :)

https://www.grammarly.com/